guides/release/models/creating-updating-and-deleting-records.md

```diff
@@ -118,7 +118,7 @@ let post = store.createRecord('post', {
 let self = this;
 
 function transitionToPost(post) {
-  self.transitionToRoute('posts.show', post);
+  this.router.transitionTo('posts.show', post);
```

Comment:
The surrounding code initially didn't make sense to me because of a false indentation. I think it'd be nice to use newer JS syntax as well, to make the code more readable.

Could we update the code block to:

```js
// Ember Data assumed
const newPost = this.store.createRecord('post', {
  title: 'Rails is Omakase',
  body: 'Lorem ipsum'
});

try {
  await newPost.save();
  this.router.transitionTo('posts.show', newPost.id);
} catch (error) {
  // Handle the error
}
```